CONTRIBUTING.md

```diff
@@ -13,11 +13,14 @@ as well as on merge to `main`.
 ### Local Testing
 
 ```sh
-# Unit tests
-./gradlew :library:unitTest
+# Run setup to set the environment, including postgres running in docker:
+./scripts/setup
```

Comment:
see comment in `service/README.md` below - happy to consolidate conversation in that thread (commenting to bookmark instance)